docs/docs-site/content/releases/release-notes-2.0.0-beta.md

```diff
@@ -365,7 +365,7 @@ List of All Commits
 * HUE-633. [jobsub] Port examples to new design models
 * [test] Work around a webhdfs redirect bug in test
 * [doc] document dependency on libldap2-dev
-* HUE-648. [fb] filebrowser:filebrowser.views.view() makes too many filesystem calls
+* HUE-648. [fb] filebrowser.views.view() makes too many filesystem calls
```

Comment:
4 docs changes to avoid as not true at the time

Reply:
ping

Reply:
what thing I have to do here I am not understanding? @romainr